src/main/java/ru/skypro/homework/controller/AdController.java

```diff
@@ -56,26 +58,33 @@ public ResponseEntity<Ads> adInfo(Authentication authentication) {
     //***
 
     @PostMapping(consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
-    public ResponseEntity<?> newAd(@RequestParam("properties") Ad ad, @RequestParam("image") MultipartFile picture) {
-        adService.newAd(ad, String.valueOf(picture));
-        return ResponseEntity.ok().build();
+    public ResponseEntity<?> newAd(@RequestParam(required = false, name = "properties") CreateOrUpdateAd createOrUpdateAd, @RequestParam("image") MultipartFile picture, Authentication authentication) {
```

Comment:
```suggestion
    public ResponseEntity<?> newAd(@RequestParam(name = "properties") CreateOrUpdateAd createOrUpdateAd, @RequestParam("image") MultipartFile picture, Authentication authentication) {
```
зачем required = false? это обязательная часть, сейчас у тебя может быть она null и поэтому ошибка в сервисе